src/modules/global/modal-additional-info/ModalAdditionalInfo.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <base-modal @close-modal="closeModal">
+    <div class="flex flex-col">
+      <section class="mb-2">
+        <h1 class="font-heading text-3xl">Additional Information</h1>
+        <p class="text-gray-5">
+          We would like to know more about you!
+        </p>
+      </section>
+      <form @submit.prevent="validateForm">

```

Comment:
why we only validate only on submit, this should happen at realtime

Reply:
Wait for UI